a.py

```diff
@@ -1,92 +1,101 @@
 from scapy.all import sniff, IP
 import threading
 import time
-import socket
 import os
 import ipaddress
+import clipboard
 
+# Defining import variables
 traffic_data = {}
 lock = threading.Lock()
+print('Starting Lacakgrak... Output will appear in 10 seconds.')
 
 def packet_callback(packet):
+    """
+    Callback function that processes network packets captured by the sniffer.
+    
+    This function is called for each network packet captured by the sniffer. It extracts the source and destination IP addresses from the packet, and updates the `traffic_data` dictionary with the total bytes sent and received for each IP address. The `lock` object is used to ensure thread-safe access to the `traffic_data` dictionary.
+    """
     if IP in packet:
+        global ip_dst

```

Comment:
The use of global variable in python is considered bad practice and should generally be avoided.

https://stackoverflow.com/questions/19158339/why-are-global-variables-evil